filters/mutable.filter.ts

```diff
@@ -0,0 +1,28 @@
+import { Filter, FilterResult } from './pool-filters';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { LiquidityPoolKeysV4 } from '@raydium-io/raydium-sdk';
+import { getPdaMetadataKey } from '@raydium-io/raydium-sdk';
+import { getMetadataAccountDataSerializer, MetadataAccountData } from '@metaplex-foundation/mpl-token-metadata';
```

Comment:
@metaplex-foundation/mpl-token-metadata should be added to package.json

Reply:
added @metaplex-foundation/mpl-token-metadata to package.json